tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -157,7 +157,7 @@ public function testCreateTableWithForeignKeys()
 
         $table = new \Phinx\Db\Table('ntable', array(), $this->adapter);
         $table->addColumn('realname', 'string')
-              ->addColumn('tag_id', 'integer')
+              ->addColumn('tag_id', 'integer', array('signed' => false))
```

Comment:
Just going off the docs, isn't 'signed' ignored outside of MySQL anyway? Certainly needs further input.